api_retry/util.py

```diff
@@ -0,0 +1,86 @@
+# standard libraries
```

Comment:
It could. I'm not sure I want to make this project's on-time release any more dependent than we already are on updating that library (I already need @pushyami's fix from February, and some dependency updates would be nice).

Reply:
OK. Can put off to later time after the summer release.